tools/tokenserver/purge_old_records.py

```diff
@@ -45,6 +45,7 @@ def purge_old_records(
     dryrun=False,
     force=False,
     override_node=None,
+    uid_range=None,
```

Comment:
`uid_range` is never used in this method (it's not passing `range` to `get_old_user_records`)

Reply:
Argh. Missed that when doing the clean-up. Thanks.